packages/dependency-extraction-webpack-plugin/util.js

```diff
@@ -1,4 +1,5 @@
 const WORDPRESS_NAMESPACE = '@wordpress/';
+const BUNDLED_PACKAGES = '@wordpress/icons';
```

Comment:
I think you meant to use an array of packages here:

```suggestion
const BUNDLED_PACKAGES = [ '@wordpress/icons' ];
```

---

This seems to only be relevant for packages in the `@wordpress` namespace (if we want to bundle other packages, they wouldn't be included below). We might lest those and handle them as needed:

```suggestion
const BUNDLED_PACKAGES = [ 'icons' ];
```

and changes below would become:

```js
	if ( request.startsWith( WORDPRESS_NAMESPACE ) ) {
		const packageName = request.substring( WORDPRESS_NAMESPACE.length );
		if ( BUNDLED_PACKAGES.includes( packageName ) ) {
			return undefined;
		}
		return [ 'wp', camelCaseDash( packageName ) ];
	}
```

Reply:
This might trip any package that has the word `icons` in it.